enclave/server.ts

```diff
@@ -166,6 +203,53 @@ io.on("connection", (socket: Socket) => {
     socket.on("decrypt", (l: Location, pubkey: string, sig: string) => {
         decrypt(socket, l, Player.fromPubString(pubkey), sig);
     });
+
+    nStates.on(nStates.filters.NewMove(), (hUFrom, hUTo) => {
+        console.log("hUFrom", hUFrom);
+        console.log("hUTo: ", hUTo);
+
+        hUFrom = Utils.intoBigNumber(hUFrom);
+        hUTo = Utils.intoBigNumber(hUTo);
+
+        for (let i = 0; i < claimedMoves.length; i++) {
```

Comment:
could we make claimedMoves a map for fast indexing

Reply:
same question as above: what should the mapping look like?